### ISSUE TITLE
NFTs: "mint as NFT" button is nowhere near obvious enough

### ISSUE THREAD

**Issue by mochet** [Joystream/atlas#6387]
# Problem
TLDR: the "mint as NFT" option is way too obscured visually. Here is my experience:

I am uploading a video and am intending to make it into an NFT (this is already my mission here and specifically why I am uploading this particular video). I get this helpful pop up:
![image](https://github.com/Joystream/atlas/assets/50830161/46c09416-5488-4100-aeeb-2353b69e0437)

I then press `"Ok, got it!"`

This is now what I see:
![image](https://github.com/Joystream/atlas/assets/50830161/6ddbd9b5-d1a6-4bc0-8a9a-7bfdca73de2f)

I can't find any obvious way to mint an NFT--so I search around the screen and I try the following:
* clicking the video-- this opens it as a player
* I try clicking the `blue pencil icon`--this opens up the editor, but there is still no option/mention of NFTs
![image](https://github.com/Joystream/atlas/assets/50830161/7c913ec7-bb09-4a32-abfa-b4d51e8153e6)
* I then scan the left sidebar--but there is still no NFT option
![image](https://github.com/Joystream/atlas/assets/50830161/75990a1e-c018-4a83-a171-d366027bc5e6)
* I click around a few other sections like Channel/Uploads and still can't find anything

* I finally realize that there is a `3 dot menu` next to the video, which I click and then finally see the "mint as NFT" option
![image](https://github.com/Joystream/atlas/assets/50830161/2fe29ee6-f908-49c0-9a64-69593c2c701c)

The core problem here is:
* No mention of NFTs visually anywhere in the studio, as a creator I expect to see this somewhere since Joystream has NFTs
* The editor doesn't feature a "mint as NFT" option--this would be the most logical place I would expect to find it
* The `3 dot menu` isn't visible unless a user does a mouseover--I do not know if mobile users even can see this?

# Solutions
I would suggest some combination of the following:
* Add an `NFT` button under the `blue pencil icon`--maybe it defaults to grey if the video isn't an NFT?
* Add an `NFT` sidebar button, such that there is always `NFT` visually displayed at all times
* Add an `NFT` tab or something within the editor
* Make the `3 dot menu` more prominent by default? On YT it is always visible when viewing anyone's channel (but not on the homepage--but not within their studio offering in which case you have to hover over,. _but_ it does show multiple buttons when mouseover'd)
